src/Test.Utilities/AdditionalMetadataReferences.cs

```diff
@@ -89,7 +89,7 @@ private static ReferenceAssemblies CreateDefaultReferenceAssemblies()
             referenceAssemblies = referenceAssemblies.AddAssemblies(ImmutableArray.Create("System.Xml.Data"));
 #endif
 
-            referenceAssemblies = referenceAssemblies.AddPackages(ImmutableArray.Create(new PackageIdentity("Microsoft.CodeAnalysis", "3.0.0")));
+            referenceAssemblies = referenceAssemblies.AddPackages(ImmutableArray.Create(new PackageIdentity("Microsoft.CodeAnalysis", "3.6.0")));
```

Comment:
Note: I tried using 3.8.0, but I got version conflicts with System.Collections.Immutable, so I went with the lowest version containing source generator stuff.

Reply:
💡 It might be better to just have a separate `ReferenceAssemblies` instance for source generator testing.

Reply:
This seems the best path forward imo. When I was originally hitting the issue I didn't realize you could just specify the version and it would retrieve it.

Reply:
I see that @ryzngard already doing that in https://github.com/dotnet/roslyn-analyzers/pull/5235. Going to close the PR.